pkgs/os-specific/linux/lm-sensors/default.nix

```
@@ -18,8 +18,12 @@ stdenv.mkDerivation rec {
   buildInputs = [ perl ]
    ++ stdenv.lib.optional sensord rrdtool;
 
-  makeFlags = [ "PREFIX=${placeholder "out"}" "ETCDIR=${placeholder "out"}/etc" ]
-    ++ stdenv.lib.optional sensord "PROG_EXTRA=sensord";
+  makeFlags = [
+    "PREFIX=${placeholder "out"}"
+    "ETCDIR=${placeholder "out"}/etc"
+    "CC=${stdenv.cc.targetPrefix}gcc"
```

```suggestion
    "CC=${stdenv.cc.targetPrefix}cc"
```
or it won't work with clang

---

Fixed.